An update to episode 'collaborating' (#31)
carpentries/lesson-development-training: episodes/24-collaborating.md

Collaborators can also use other platforms to discuss lesson development or receive contributions from newcomers who are not yet fluent in using GitHub's systems of communication. The Carpentries can assist with creating a mailing list specific to the development of your lesson on their [TopicBox](https://carpentries.topicbox.com/) platform for managing threaded email discussions. Also make sure to join the [Incubator lesson developers mailing list](https://carpentries.topicbox.com/groups/incubator-developers) on TopicBox to keep an eye on announcements and discussions relating to lesson development in general within the Carpentries community.

> Meeting minutes are a useful way of providing a permanent record of the purpose of a meeting and what was talked about, including any decisions made or actions taken, that can be referred back to for any follow-ups. Recording meeting minutes does not have to be hard - remember that you are part of a team and there are many platforms that allow collaborative note taking. The Carpentries provide [CodiMD](https://codimd.carpentries.org/) and [Etherpad](https://pad.carpentries.org/) instances as two options for taking shared notes, widely used at Carpentries workshops for sharing pieces of code, references and notes between insturcors and learners. For some advice on holding effective meetings, see the [“Meetings, meetings, meetings”](http://teachtogether.tech/en/index.html#s:meetings) section of [Teaching Tech Together](http://teachtogether.tech/en/index.html) by Greg Wilson.

Reviewer: ```suggestion
Meeting minutes are a useful way of providing a permanent record of the purpose of a meeting and what was talked about, including any decisions made or actions taken, that can be referred back to for any follow-ups. Recording meeting minutes does not have to be hard - remember that you are part of a team and there are many platforms that allow collaborative note taking. The Carpentries provides [CodiMD](https://codimd.carpentries.org/) and [Etherpad](https://pad.carpentries.org/) instances as two options for taking shared notes, widely used at workshops for sharing pieces of code, references and notes between instructors and learners. For some advice on holding effective meetings, see [the _Meetings, meetings, meetings_ section of Teaching Tech Together](http://teachtogether.tech/en/index.html#s:meetings) by Greg Wilson.
```

Once your project board has a large number of cards on it, you might want to begin priorisiting them. Not all tasks are going to be equally important, and some will require others to be completed before they can even be begun. Common methods of prioritisation include:

- **Vertical position**: the vertical arrangement of cards in a column implicitly represents their importance. High-priority bugs go to the top of `To Do`, whilst tasks that depend on others go beneath them. This is the easiest one to implement, though you have to remember to correctly place cards when you add them.
- **Priority columns**: instead of a single `To Do` column, you can have two or more, for example - `To Do: Low Priority` and `To Do: High Priority`. When adding a card, you pick which is the appropriate column for it. You can even add a Triage column for newly-added issues that you’ve not yet had time to classify. This format works well for project boards devoted to bugs.

> - **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Label colours can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label. You also need to be aware of accessibility issues for colour-blind people if choosing this route for issue prioritisation. 

Author: ```suggestion
- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Label colours can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label. If you choose this route for issue prioritisation - be aware of accessibility issues for colour-blind people when picking colours. 
```

Here are some aspects of governing a project that you should consider. These are borrowed from [the _Working in teams_ chapter](https://merely-useful.tech/py-rse/teams.html) of [_Research Software Engineering with Python_](https://merely-useful.tech/py-rse/index.html), a book on how to work productively in small teams where everyone is welcome:

> - [establishing a Code of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)

Reviewer: ```suggestion
- [Codes of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
```

I think this is more consistent with the phrasing of the other points in the list.
